g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/common/CLDRConstants.java

```diff
@@ -47,6 +50,7 @@
 	public static final String ALL_CATEGORIES = "dates,numbers,plurals,measurements,currencies,dateFields";
 	public static final String LOCALE_TERRITORIES_PATH = "cldr/localedata/{0}/territories.json";
 	public static final String LOCALE_LANGUAGES_PATH = "cldr/localedata/{0}/languages.json";
+	public static final String CONTEXT_TRANSFORM_PATH = "cldr/localedata/{0}/contextTransforms.json";
```

Comment:
It's not reasonable to put contextTransform data in localedata folder, since it not only contains locale context data, but also other context data, such as that for date. So I think put it into a parallel folder with 'localedata' is better.

Reply:
fixed